src/fragmentarium/ui/display/Transliteration.tsx

```diff
@@ -143,16 +151,60 @@ function DisplayLine({
   )
 }
 
+function DollarLineAndAtLine({
```

Comment:
It is not a good idea to make this kind of combo function. All the lines have a different model and you will end up with a huge if-else-block inside. You already have the logic below the select the correct component type.

If you find a common denominator and want to make a component based on that, name it accordingly.

Reply:
You split the card to ruling and others, so it does not make sense to care about other types of lines here.